client/exampleData.js

```diff
@@ -1,27 +1,27 @@
-const exampleData = {
-  protection_plan : {
-    exists : true,
-    name : 'consequuntur repellendus saepe',
-    price : 182,
-    years : 5,
-    provider : "Gulgowski Inc",
-    rating : 4,
-    description : "Et non id. Laudantium quae dolorum dolorem quia debitis explicabo quia fuga assumenda. Sed natus alias delectus. Amet incidunt quae velit dolores et eaque adipisci."  
+const data = {
+  protection_plan: {
+    exists: true, // (GOOD)
+    name: 'consequuntur repellendus saepe', // (GOOD)
+    price: 182, // <-- not too sure what digits place to set this to (GOOD)
```

Comment:
looks like amazon renders whole dollar amounts with ".00" at the end if that's what you're asking? https://www.amazon.com/gp/product/B00XQC11RA